packages/manager/modules/carbon-calculator/src/dashboard/carbon-footprint/controller.js

```diff
@@ -55,6 +63,11 @@ export default class CarbonFootprintCtrl {
   $onInit() {
     this.$asyncFetching = null;
     this.previousMonth = this.carbonFootprintService.computePreviousMonth();
+    this.availableServices = SERVICES_AVAILABLE.map((u) =>
+      this.$translate.instant(
+        `carbon_calculator_dashboard_billing_explanation_${u}`,
+      ),
+    );
```

Comment:
```suggestion
    this.availableServices = SERVICES_AVAILABLE.map((service) =>
      this.$translate.instant(
        `carbon_calculator_dashboard_billing_explanation_${service}`,
      ),
    );
```
It's better to have a variable name